package.json

```diff
@@ -34,44 +34,44 @@
   },
   "homepage": "https://github.com/tablelandnetwork/eth-tableland#readme",
```

Comment:
I've run an `npm update`, because for some reason the hook only worked in the latest openzeppelin impl. lmk if that's ok.

Reply:
I _think_ this is ok... but it might make upgrading the contract more difficult. But since these are all changes on the implementation contract side, we're _probably_ ok. We'll want to create an upgrade test to be sure.